mantle/cmd/ore/aliyun/copy-image.go

```diff
@@ -38,13 +38,15 @@ After a successful run, the final line of output will be a line of JSON describi
 	sourceImageID        string
 	destImageName        string
 	destImageDescription string
+	waitForReady         bool
 )
 
 func init() {
 	Aliyun.AddCommand(cmdCopyImage)
 	cmdCopyImage.Flags().StringVar(&sourceImageID, "image", "", "source image")
 	cmdCopyImage.Flags().StringVar(&destImageName, "name", "", "destination image name")
 	cmdCopyImage.Flags().StringVar(&destImageDescription, "description", "", "destination image description")
+	cmdCopyImage.Flags().BoolVar(&waitForReady, "wait-for-ready", false, "wait for the copied image to be marked available")
```

Comment:
```suggestion
	cmdCopyImage.Flags().BoolVar(&waitForReady, "wait-for-ready", true, "wait for the copied image to be marked available")
```

thoughts on making it the default?

Reply:
WFM

Reply:
Actually, if we make this `true` by default, its a default behavior change and may affect things that are consuming the command and have expectations on how it works.  (It's very unlikely we have other consumers besides our own pipelines, but I think I want to err on the side of caution here)

Reply:
I'd just wonder what consumers would want the command to return before the image was available (i.e. what is the sane default behavior?), but yield to your judgement here.